domino_environments/_environments.py

```diff
@@ -124,9 +137,13 @@ def __init__(self, host=None, api_key=None, domino_token_file=None):
         """
         self._configure_logging()
 
-        host: str = clean_host_url(get_host_or_throw_exception(host))
-        domino_token_file = get_path_to_domino_token_file(domino_token_file)
-        api_key: str = get_api_key(api_key)
+        _host = host or os.getenv(DOMINO_HOST_KEY_NAME)
+        assert _host, ("Host must be supplied as a parameter or through the "
```

Comment:
please raise an error instead of an assert, it's easier to handle exceptions that it is asserts.

Reply:
@mildmelon i was copying this piece directly from https://github.com/dominodatalab/python-domino/blob/0c4fe25eeb83a51d6ea987c9fb9c4656ba292822/domino/domino.py#L27 , but i can change it to an exception

Reply:
eh it's fine, I'd prefer consistency with the `python-domino` library